hack/make/.binary

```diff
@@ -56,6 +56,9 @@ go build \
 		$LDFLAGS_STATIC_DOCKER
 	" \
 	$SOURCE_PATH
+
+    # i don't know what i'm doing in these files
+    gcc -o "$DEST/docker-init" -static init/init.c
```

Comment:
nit:
- indentation seems unconsistent
- I think this compilation should be done only when `uname -s` is `Linux`